bodyAgent/.htaccess

```
@@ -0,0 +1,8 @@
+# Ativa o mecanismo de rewrite
+RewriteEngine On
+
+# Redireciona a URI principal para o arquivo TTL no GitHub
+RewriteRule ^$ https://raw.githubusercontent.com/IderliSouza/w3id.org/main/bodyAgent/bodyAgent.ttl [R=303,L]
```

Align with URL in README.
```suggestion
RewriteRule ^$ https://raw.githubusercontent.com/IderliSouza/bodyAgent-Ontology/refs/heads/main/bodyAgent.ttl [R=303,L]
```